src/Features/Core/Portable/CodeRefactorings/CodeRefactoringService.cs

```diff
@@ -180,10 +182,10 @@ private ConcatImmutableArray<CodeRefactoringProvider> GetProviders(Document docu
                         // Serialize access for thread safety - we don't know what thread the refactoring provider will call this delegate from.
                         lock (actions)
                         {
-                            // Add the Refactoring Provider Name to the parent CodeAction's CustomTags.
+                            // Add the Refactoring Provider Name to the parent CodeAction's Tags.
                             // Always add a name even in cases of 3rd party refactorings that do not export
                             // name metadata.
-                            action.AddCustomTag(providerMetadata?.Name ?? provider.GetTypeDisplayName());
+                            action.AddTag($"{ProviderTagPrefix}{providerMetadata?.Name ?? provider.GetTypeDisplayName()}");
```

Comment:
Note with these prefixes, Razor may no longer need to use reflection to distinguish code fix names from other tags. cc/ @NTaylorMullen